app/assets/javascripts/views/story_view.js

```diff
@@ -784,11 +784,23 @@ module.exports = FormView.extend({
         />,
         $noteForm.get(0)
       );
-
-      $noteForm.find('textarea').atwho({
+      const addNoteButton = $noteForm.find('button')
```

Comment:
Looks like we have some duplication here and in the task form, can we extract the logic and reuse it just passing the form instance?